crates/orchestrator/src/queue/job_queue.rs

```diff
@@ -102,14 +117,47 @@ enum DeliveryReturnType {
     NoMessage,
 }
 
-pub async fn add_job_to_process_queue(id: Uuid, config: Arc<Config>) -> EyreResult<()> {
+pub async fn add_job_to_process_queue(id: Uuid, job_type: &JobType, config: Arc<Config>) -> EyreResult<()> {
     tracing::info!("Adding job with id {:?} to processing queue", id);
-    add_job_to_queue(id, JOB_PROCESSING_QUEUE.to_string(), None, config).await
+
+    match job_type {
```

Comment:
I think we can simplify this by creating a trait here that matches jobType to it's process/verify queue name.

```RUST 
trait QueueName {
    fn process_queue_name(&self) -> String;
    fn verify_queue_name(&self) -> String;
}

impl QueueName for JobType {
    fn process_queue_name(&self) -> String {
        match self {
            JobType::SnosRun => SNOS_JOB_PROCESSING_QUEUE,
            JobType::ProofCreation => PROVING_JOB_PROCESSING_QUEUE,
            // so on . . .           
        }.to_string()
    }
    fn verify_queue_name(&self)- > String {
        // similar
    }
}
```

Then you can just do :
`add_job_to_queue(id, job_type.process_queue_name(), None, config).await;`
inside 
`add_job_to_process_queue` or `add_job_to_verify_queue`


This looks cleaner to me 
WDYT @byteZorvin ?

Reply:
Agreed updated !!